src/backend/distributed/executor/insert_select_executor.c

```diff
@@ -419,25 +419,6 @@ PartitionColumnIndexFromColumnList(Oid relationId, List *columnNameList)
 }
 
 
-/*
- * IsSupportedRedistributionTarget determines whether re-partitioning into the
- * given target relation is supported.
- */
-bool
-IsSupportedRedistributionTarget(Oid targetRelationId)
```

Comment:
Minor naming note:

I'd rather keep this function over inlining into the `IsRedistributablePlan`.

`IsRedistributablePlan` deals only with the `plan`, now we also check for the target table. It feels slightly simpler to follow these two concepts separately.

Reply:
reintroduced.